x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_preview_section.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import type { OnTimeChangeProps } from '@elastic/eui';
+import {
+  EuiAccordion,
+  EuiFormRow,
+  EuiPanel,
+  EuiSpacer,
+  EuiSuperDatePicker,
+  EuiTitle,
+  EuiCallOut,
+  EuiButton,
+  EuiIcon,
+  EuiText,
+} from '@elastic/eui';
+import { RiskScoreEntity } from '../../../common/risk_engine/types';
+import { RiskScorePreviewTable } from './risk_score_preview_table';
+import * as i18n from '../translations';
+import type { RiskScore } from '../../../server/lib/risk_engine/types';
+import { useRiskScorePreview } from '../api/hooks/use_preview_risk_scores';
+
+interface IRiskScorePreviewPanel {
+  showMessage: string;
+  hideMessage: string;
+  isLoading: boolean;
+  items: RiskScore[];
+  type: RiskScoreEntity;
+}
+
+const getRiskiestScores = (scores: RiskScore[] = [], field: string) =>
+  scores
+    ?.filter((item) => item?.identifierField === field)
+    ?.sort((a, b) => b?.totalScoreNormalized - a?.totalScoreNormalized)
+    ?.slice(0, 5) || [];
+
+const RiskScorePreviewPanel = ({
+  items,
+  showMessage,
+  hideMessage,
+  isLoading,
+  type,
+}: IRiskScorePreviewPanel) => {
+  const [trigger, setTrigger] = useState<'closed' | 'open'>('open');
+  const onToggle = (isOpen: boolean) => {
+    const newState = isOpen ? 'open' : 'closed';
+    setTrigger(newState);
+  };
+
+  return (
+    <EuiPanel hasBorder={true}>
+      <EuiAccordion
+        initialIsOpen={true}
+        isLoading={isLoading}
+        id={'host-table'}
+        buttonContent={trigger === 'closed' ? showMessage : hideMessage}
+        forceState={trigger}
```

Comment:
why do we need forceState here? 
I assume this is the reason for using trigger as string and not boolean?

Reply:
To be able to[control](https://elastic.github.io/eui/#/layout/accordion#controlling-toggled-state) accordion state and change the message